### ISSUE TITLE
Cache issues?

### ISSUE THREAD

**Issue by trebitzki** [aymericb/exposition#3]
I rotated some of the photos in my album and loaded update.html to refresh the cache. The photos are still unrotated. I even cleared the cache manually - no go. I suppose the rotated image is cached internally inside the image, so exposition doesn't see it? When I open the image with Preview and do a Save As... the new image is seen correctly (rotated) in exposition.

So I guess it's just something the user has to know about.

**Comment by aymericb** [comment]
I think this is probably the browser cache. I set an expiration of 1hr in `PHOTO_CACHE_DURATION` inside [config.php](https://github.com/aymericb/exposition/blob/master/php/core/config.php). Try to clear the cache of the browser, use a different browse, come back after one hour, etc...

**Comment by trebitzki** [comment]
No, loading the site in a different browser - Chrome rather than Firefox - still shows un-rotated pictures.

**Comment by aymericb** [comment]
Not sure what's going on here. I use [filemtime](http://php.net/manual/en/function.filemtime.php) to compare the _modification date_ of the file in the cache and the original. If the original has been changed more recently, then it's the one I use and regenerate the cache file. It's done in [core/image.php](https://github.com/aymericb/exposition/blob/master/php/core/image.php).

**Comment by trebitzki** [comment]
I really think it's not an issue with Exposition. As I said, I had cleared the cache.
